src/neo4j_graphrag/experimental/pipeline/pipeline.py

```diff
@@ -386,6 +390,12 @@ def draw(
         G.draw(path)
 
     def get_pygraphviz_graph(self, hide_unused_outputs: bool = True) -> pgv.AGraph:
+        if pgv is None:
+            raise ImportError(
+                "Could not import pygraphviz. "
+                "Follow installation instruction in pygraphviz documentation "
```

Comment:
Could be useful to add a link here

Reply:
Not a big fan of adding links, it's already hard for us to update links to our own domain without adding the pain to maintain links to outside of our org :)

Reply:
Perhaps a stackoverflow thread that discusses this? It's unlikely that will become outdated.

Reply:
Imo anything to smoothen the user experience is useful here given encountering this installation issue is common

Reply:
Well, I also think developers should be able to google such things :) But added in the README [here](https://github.com/neo4j/neo4j-graphrag-python/pull/137/commits/b2c0f971eb9d4f6b85a5679cbbbf35d09aebdcff)